### PR TITLE
support custom triple metadata & transform pressure mark

### PR DESCRIPTION
**Pull request by OrezzerO**
### Motivation:

Explain the context, and why you're making that change.
To make others understand what is the problem you're trying to solve.

### Modification:

Describe the idea and modifications you've done.

### Result:

Fixes #<GitHub issue number>. 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

File: remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/filter/PressureMarkTransformFilter.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    public static final String MARK     = "mark";
+    public static final String T        = "T";
+
+    @Override
+    public SofaResponse invoke(FilterInvoker invoker, SofaRequest request) throws SofaRpcException {
+        // consumer side, if in provider side,loadTest always false
+        SofaTracerSpan currentSpan = SofaTraceContextHolder.getSofaTraceContext().getCurrentSpan();
+        boolean loadTest = TracerUtils.isLoadTest(currentSpan);
+        if (loadTest) {
+            RpcInvokeContext.removeContext();
```
Comment (loull521) [inline remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/filter/PressureMarkTransformFilter.java]: 这个看着有点奇怪

**Review by NeGnail — APPROVED** [review]
LGTM

File: remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/filter/PressureMarkTransformFilter.java (hunk 1 above)
Comment (OrezzerO) [reply remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/filter/PressureMarkTransformFilter.java]: fixed

**Review by alaneuler — APPROVED** [review]
LGTM

**Pull request merged**